commit 0e4fa1cbdf32ca0767f8cd92ec9ced89f7760158
Author: Aneesh Kumar K.V <aneesh.kumar@linux.vnet.ibm.com>
Date:   Tue Feb 7 00:09:27 2017 +0530

    powerpc/mm: Add MMU_FTR_KERNEL_RO to possible feature mask
    
    commit a5ecdad4847897007399d7a14c9109b65ce4c9b7 upstream.
    
    Without this we will always find the feature disabled.
    
    Fixes: 984d7a1ec6 ("powerpc/mm: Fixup kernel read only mapping")
    Signed-off-by: Aneesh Kumar K.V <aneesh.kumar@linux.vnet.ibm.com>
    Acked-by: Balbir Singh <bsingharora@gmail.com>
    Signed-off-by: Michael Ellerman <mpe@ellerman.id.au>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1354_0e4fa1cbdf32ca0767f8cd92ec9ced89f7760158.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1354_0e4fa1cbdf32ca0767f8cd92ec9ced89f7760158.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
